staging/src/k8s.io/client-go/rest/with_retry.go

```diff
@@ -354,6 +354,10 @@ func retryAfterResponse() *http.Response {
 }
 
 func retryAfterResponseWithDelay(delay string) *http.Response {
+	return retryAfterResponseWithCodeAndDelay(http.StatusInternalServerError, delay)
+}
+
+func retryAfterResponseWithCodeAndDelay(code int, delay string) *http.Response {
```

Comment:
I guess you meant to set the StatusCode based on the input var here, right?

Reply:
oops, typo, fixed.